tests/integration/testdata/buildcmd/Go/.gitignore

```diff
@@ -0,0 +1 @@
+go.sum
```

Comment:
Should this be in the base .gitignore? Will git even pick this up for the repo?

Reply:
Git does pick it up, but I can move it to the base of the repo.

Reply:
O cool. I didn't actually know that. I don't have a preference where it lives, just didn't realize git was smart like that :)